.werft/platform-delete-preview-environments-cron.ts

```diff
@@ -77,8 +133,5 @@ function expectedNamespaceFromBranch(branch: string): string {
 }
 
 function parseBranch(branch: string): string {
```

Comment:
parseBranch can be deleted now as it is no longer needed.

Reply:
Done